auth/rand/random.go

```diff
@@ -0,0 +1,43 @@
+package rand
+
+import (
+	"crypto/rand"
+
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	chars = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
+
+	randRead = rand.Read
+)
+
+func init() {
+	// exits with 1 when crypto/rand is not available
+	Token(1)
```

Comment:
Is there no other way of checking if a package isn't available?
Couldn't there be another exception or reason why the `randRead` threw an error?